apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/config.py

```diff
@@ -110,12 +112,31 @@
 cfg.CONF.register_opts(apic_opts, "ml2_cisco_apic")
 
 
+def _parse_files(conf_files):
+    try:
+        multi_parser = cfg.MultiConfigParser()
+        multi_parser.read(conf_files)
+        return multi_parser.parsed
+    except AttributeError:
+        # Oslo 6.0.0+
+        sections = {}
+        for filename in conf_files:
+            parser = cfg.ConfigParser(filename, sections)
+            try:
+                parser.parse()
+            except IOError:
+                continue
```

Comment:
this deserves a log message as well, no?

Reply:
Sure -- will ad.